tests/test_Pipelines/test_skypy_pipeline.py

```diff
@@ -8,6 +9,30 @@ def setup_method(self):
         sky_area = Quantity(value=0.05, unit="deg2")
         self.pipeline = SkyPyPipeline(skypy_config=None, sky_area=sky_area)
 
+    def test_cosmology_initialization(self):
+        # LambdaCDM model
+        cosmo1 = LambdaCDM(H0=70, Om0=0.3, Ode0=0.7)
+        pipeline1 = SkyPyPipeline(cosmo=cosmo1)
+        blue_galaxies = pipeline1.blue_galaxies
+        assert blue_galaxies[0]["z"] > 0
```

Comment:
these tests do not test whether the cosmology is overwritten. Are there other test one can do to test a specific cosmology-dependent value?